css/selectors/is.json

```diff
@@ -2,219 +2,276 @@
-                "notes": "Combinators in the selector list argument may not match correctly (see <a href='https://crbug.com/842157'>bug 842157</a>)."
-              },
-              {
-                "version_added": "66",
-                "version_removed": "71",
-                "alternative_name": ":matches()",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-web-platform-features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Combinators in the selector list argument may not match correctly (see <a href='https://crbug.com/842157'>bug 842157</a>)."
-              },
-              {
-                "version_added": "12",
-                "alternative_name": ":-webkit-any()",
-                "notes": "Doesn't support combinators."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "66",
-                "version_removed": "71",
-                "alternative_name": ":matches()",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-web-platform-features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Combinators in the selector list argument may not match correctly (see <a href='https://crbug.com/842157'>bug 842157</a>)."
-              },
-              {
-                "version_added": "18",
-                "alternative_name": ":-webkit-any()",
-                "notes": "Doesn't support combinators."
-              }
-            ],
-            "edge": [
-              {
-                "version_added": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-web-platform-features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Combinators in the selector list argument may not match correctly (see <a href='https://crbug.com/842157'>bug 842157</a>)."
-              },
-              {
-                "version_added": "79",
-                "alternative_name": ":-webkit-any()",
-                "notes": "Doesn't support combinators."
-              }
-            ],
-            "firefox": [
-              {
-                "version_added": "78"
-              },
-              {
-                "version_added": "77",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.is-where-selectors.enabled",
-                    "value_to_set": "enabled"
-                  }
-                ],
-                "notes": "Enabled by default in Firefox Nightly."
-              },
-              {
+        "basic_support": {
```

Comment:
You're right to create a new feature for `forgiving_selector_list`, but the existing `"__compat"` object doesn't need to be wrapped in this new `"basic_support"` object. Instead, put the original `"__compat"` key and the new `"forgiving_select_list"` key at the same level (with `"forgiving_select_list"` after the original `"__compat"`).